### ISSUE TITLE
Allow change of label "new selection" 

### ISSUE THREAD

**Issue by michalzubkowicz** [ericgio/react-bootstrap-typeahead#122]
Please allow change of label "new selection" or create another way to change language

**Comment by ericgio** [comment]
Use the `newSelectionPrefix` prop

**Comment by michalzubkowicz** [comment]
Thank you. That wasn't documented.

**Comment by ericgio** [comment]
Actually it is documented, both on Github & NPM: https://github.com/ericgio/react-bootstrap-typeahead/blob/master/docs/Props.md

**Comment by michalzubkowicz** [comment]
You're right. I've seen that option but missunderstood description. Sorry

**Comment by Joelfo** [comment]
The Typeahead component now have its own newSelectionPrefix attribute, which is great and works fine! But it not seems to be doccumented in the API.md file on the docs folder. I was having some problems trying to use the TypeaheadMenu (maybe I didn't understand how to use it well) and the new attribute helped me a lot, I discovered it in the CustomSelection example which is also great. So if anyone come at this issue trying to change the New Selection Label as I did, just use the newSelectionPrefix attribute on the Typeahead component.
